docs/topics/sgx.md

```diff
@@ -3,15 +3,28 @@
 [Intel&reg; Secure Guard Extension](https://software.intel.com/en-us/sgx) (Intel&reg; SGX) is an architecture extension designed to increase the security of application code and data.
 Developers may choose [Intel&reg; SGX SDK](https://software.intel.com/en-us/sgx-sdk) or [Open Enclave SDK](https://github.com/Microsoft/openenclave/) to create applications that leverage this technology.
 
-Azure supports provisioning of SGX-enabled VMs under the umbrella of Azure Confidential Compute (ACC). You can create a Kubernetes cluster with one or multiple agent pool(s) running ACC VMs by specifying VM size `Standard_DC*` and distro `acc-16.04`.
+Azure supports provisioning of SGX-enabled VMs under the umbrella of Azure Confidential Compute (ACC). You can create a Kubernetes cluster with one or multiple agent pool(s) running ACC VMs by specifying [DC-series](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dc-series) VM size and the supported distro from the table below.
+
+| OS           | distro      | Notes |
+| ------------ | ----------- |-------|
+| Ubuntu 16.04 | `acc-16.04` | specially built image with UEFI BIOS support
+| Ubuntu 18.04 | `aks-18.04` | standard Canonical Ubuntu image
+
+The following example is a fragment of api-model file declaring two ACC agent pools, one running `Ubuntu 16.04` image on `2 vCPU` nodes, and another running `Ubuntu 18.04` image on `4 vCPU` nodes:
```

Comment:
Could you change the first part of the sentence like this:
`The following example is a fragment of a cluster definition (apimodel) file declaring...`